GVFS/GVFS.Service/Mac/GVFSService.cs

```diff
@@ -0,0 +1,115 @@
+using GVFS.Common;
+using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System;
+using System.Linq;
+using System.Threading;
+
+namespace GVFS.Service

```

Comment:
Minor: Namespace should match folder structure `namespace GVFS.Service` -> `namespace GVFS.Service.Mac`